### ISSUE TITLE
Language selection

### ISSUE THREAD

**Issue by hugoheml** [KB-RolePlay/tools-issues#98]
### Version

v2.1.31

### Description

You can't change your language

### Expected behaviour

You will be able to change your language as you want

### Reproduction steps

```bash
1. Go to https://kobralost-rp.de
2. If you have `?lang=de` in your url, change your language to french
```

**Comment by hugoheml** [comment]
Resolved in release `v2.1.33`

**Comment by hugoheml** [comment]
The `lang` query parameter is now deleted clientside.
The `locale` parameter on the store is also removed.
